packages/cli/src/lib/commands/init/processes/collect-rc-json.ts

```diff
@@ -1,7 +1,7 @@
-import { setupUrl } from '../options/url.setup';
-import { setupUfPath } from '../options/ufPath.setup';
-import { setupFormat } from '../options/format.setup';
-import { setupOutPath } from '../options/outPath.setup';
+import { setupUrl } from './url.setup';
+import { setupUfPath } from './ufPath.setup';
+import { setupFormat } from './format.setup';
+import { setupOutPath } from './outPath.setup';
```

Comment:
These imports are changed because the files did not belong in the option Directory. 

I question if we should simply inline the option files as they could all be in one declatation file and it does not necessarly make sense to split them like that!

Reply:
All of the setups have a similar structure and are used only in this file so it makes sense to have them together imo.